lib/stoplight/notifier/slack.rb

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+
+module Stoplight
+  module Notifier
+    # @see Base
+    class Slack < Base
+      # @return [Proc]
+      attr_reader :formatter
+
+      # @return [::Slack::Notifier]
+      attr_reader :slack_notifier
+
+      # @param slack_notifier [::Slack::Notifier]
+      # @param formatter [Proc, nil]
+      def initialize(slack_notifier, formatter = nil)
```

Comment:
Ah, of course. That makes the most sense. That's what the HipChat notifier does too. Let's keep it as it is.